### ISSUE TITLE
GT are all ./.

### ISSUE THREAD

**Issue by liqi814** [arq5x/lumpy-sv#366]
I have used lumpyexpress on one sample, but it has called over 1000 SVs. But the genotype of all SVs is./.

`#CHROM  POS     ID      REF     ALT     QUAL    FILTER  INFO    FORMAT  IPF3170
1       869463  1       N       <DEL>   .       .       SVTYPE=DEL;STRANDS=+-:10;SVLEN=-771;END=870234;CIPOS=-10,229;CIEND=-173,9;CIPOS95=-2,39;CIEND95=-27,3;IMPRECISE;SU=10;PE=10;SR=0        GT:SU:PE:SR     ./.:10:10:0
1       2143140 2       N       <DUP>   .       .       SVTYPE=DUP;STRANDS=-+:5;SVLEN=493;END=2143633;CIPOS=-376,9;CIEND=-10,313;CIPOS95=-73,2;CIEND95=-1,61;IMPRECISE;SU=5;PE=5;SR=0   GT:SU:PE:SR     ./.:5:5:0
1       2629079 3       N       <DEL>   .       .       SVTYPE=DEL;STRANDS=+-:4;SVLEN=-2950;END=2632029;CIPOS=-10,394;CIEND=-336,9;CIPOS95=-1,88;CIEND95=-76,2;IMPRECISE;SU=4;PE=4;SR=0 GT:SU:PE:SR     ./.:4:4:0
1       7570070 4       N       <DEL>   .       .       SVTYPE=DEL;STRANDS=+-:4;SVLEN=-1456;END=7571526;CIPOS=-10,293;CIEND=-322,9;CIPOS95=-1,71;CIEND95=-75,2;IMPRECISE;SU=4;PE=4;SR=0 GT:SU:PE:SR     ./.:4:4:0
1       10482501        5       N       <DEL>   .       .       SVTYPE=DEL;STRANDS=+-:4;SVLEN=-1267;END=10483768;CIPOS=-10,315;CIEND=-366,9;CIPOS95=-2,78;CIEND95=-85,1;IMPRECISE;SU=4;PE=4;SR=0        GT:SU:PE:SR     ./.:4:4:0
1       30569404        6       N       <DUP>   .       .       SVTYPE=DUP;STRANDS=-+:7;SVLEN=395962;END=30965366;CIPOS=-333,9;CIEND=-10,442;CIPOS95=-58,0;CIEND95=0,66;IMPRECISE;SU=7;PE=7;SR=0        GT:SU:PE:SR     ./.:7:7:0
1       29878075        7_1     N       N]1:31129382]   .       .       SVTYPE=BND;STRANDS=++:5;EVENT=7;MATEID=7_2;CIPOS=-10,350;CIEND=-10,250;CIPOS95=-2,69;CIEND95=-3,52;IMPRECISE;SU=5;PE=5;SR=0     GT:SU:PE:SR     ./.:5:5:0
1       31129382        7_2     N       N]1:29878075]   .       .       SVTYPE=BND;STRANDS=++:5;SECONDARY;EVENT=7;MATEID=7_1;CIPOS=-10,250;CIEND=-10,350;CIPOS95=-3,52;CIEND95=-2,69;IMPRECISE;SU=5;PE=5;SR=0   GT:SU:PE:SR     ./.:5:5:0
1       17684722        8       N       <DEL>   .       .       SVTYPE=DEL;STRANDS=+-:5;SVLEN=-366;END=17685088;CIPOS=-10,306;CIEND=-343,9;CIPOS95=-2,65;CIEND95=-68,2;IMPRECISE;SU=5;PE=5;SR=0 GT:SU:PE:SR     ./.:5:5:0
1       22307715        9       N       <DEL>   .       .       SVTYPE=DEL;STRANDS=+-:5;SVLEN=-24584;END=22332299;CIPOS=-10,235;CIEND=-409,9;CIPOS95=-3,45;CIEND95=-75,2;IMPRECISE;SU=5;PE=5;SR=0       GT:SU:PE:SR     ./.:5:5:0`

**Comment by ryanlayer** [comment]
Yep. You are exactly right. Lumpy makes the calls and svtyper genotypes.
